packages/interface/src/components/fullscreen-mode/style.scss

```diff
@@ -10,14 +9,65 @@ body.js.is-fullscreen-mode {
 			height: calc(100% + #{ $admin-bar-height });
 		}
 
-		#adminmenumain,
+		// Hide the top adminbar toolbar.
 		#wpadminbar {
 			display: none;
 		}
 
+		// Display the sidebar offscreen, and then transition in when needed.
+		#adminmenumain {
+			position: relative;
+			top: 60px;
+			left: -160px;
+			transition: left 0.12s ease-in-out;
+		}
+
+		// Hides the admin menu
+		// This prevents keyboard navigation from accessing inner links
+		#adminmenuwrap {
+			display: none;
+		}
+
+		#adminmenu {
+			position: relative;
+			bottom: auto;
+		}
+
+		#adminmenuback {
+			top: 60px;
+		}
+
+		.interface-interface-skeleton__body,
+		.interface-interface-skeleton__footer {
+			transition: margin-left 0.12s ease-in-out;
+		}
```

Comment:
All these `skeleton-*` styles on this stylesheet make the "full screen mode" work properly if your screen is using the Skeleton. do you think it's possible to make it independent of that? I mean no matter how the current screen is rendered, it could work?

I don't see this as a blocker but more "good to have"